cinch/roles/jenkins_master/templates/etc/nginx/conf.d/jenkins_https.conf

```diff
@@ -38,7 +41,11 @@ server {
     proxy_set_header        X-Forwarded-Proto $scheme;
     proxy_redirect http:// https://;
     proxy_pass              http://jenkins;
-  }
+    limit_req zone=jenkins burst=100 nodelay;
```

Comment:
What does this 'burst=100' number mean? Is this something that should be parameterized?